server/public/plugin/experimental/telemetry/rudder.go

```diff
@@ -0,0 +1,49 @@
+package telemetry
+
+import (
+	rudder "github.com/rudderlabs/analytics-go"
+)
+
+// rudderDataPlaneURL is set to the common Data Plane URL for all Mattermost Projects.
+// It can be set during build time. More info in the package documentation.
+var rudderDataPlaneURL = "https://pdat.matterlytics.com"
+
+// rudderWriteKey is set during build time. More info in the package documentation.
+var rudderWriteKey string
```

Comment:
Can we plan to hard-code this in the style of https://github.com/mattermost/mattermost/pull/23443? Though I realize this is running in the plugin, not the server, so we'd also need a way to get the service environment over the RPC API (not sure if the plugin has the same environment variables?), so fine to defer to another PR.

Reply:
That sounds good to me. Would you mind filing a ticket?

Reply:
Env variables are indeed not passed down to the plugin. Hence, we would need to a away to fetch the `ServiceEnvironment` via  the RPCS API.